.github/workflows/push.yml

```
@@ -15,7 +13,8 @@ jobs:
 
     steps:
       - name: Pandoc specification builder
-        uses: legreq/pandoc-spec@main
+        uses: legreq/pandoc-spec@fix-branch-deploy
```

```suggestion
        uses: legreq/pandoc-spec@main
```

This branch was merged - https://github.com/LegReq/pandoc-spec/pull/19. So main now contains the fixed action.